app/konnect/service-catalog/integrations/pagerduty.md

```diff
@@ -0,0 +1,34 @@
+---
+title: PagerDuty Integration
+content-type: reference
+config:
+    discovery_support: true
+    discovery_default: true
+    bindable_entities: "PagerDuty Service"
+    mechanism: "pull/ingestion model"
+---
+
+_Type: External_
+
+The PagerDuty integration allows you to provide a way to alert the service team (via PagerDuty services) as well as provide information on current open incidents to consumers of the service directory. 
+
+For each linked PagerDuty service, a summary should be provided showing current unresolved incidents and the current on-call user.
+An option to start an incident for the PagerDuty service
+
+## Authenticate the PagerDuty integration
+
+1. From {{site.konnect_product_name}} select **[Integrations](https://cloud.konghq.com/us/service-catalog/integrations)**. 
+2. Select **PagerDuty**, then **Install PagerDuty**.
+3. Select **Authorize**. 
+
+PagerDuty will ask you to grant consent to {{site.konnect_product_name}}. Both Read and Write scopes are required.
+
+## Bindable entities
```

Comment:
Could we call these Resources instead? "Bindable entities" is a term that our engineering team uses synonymously with Resources, and I think that we should just stick to the latter for customer-facing documentation.